pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/kitrequest/KitRequestDao.java

```diff
@@ -145,32 +121,116 @@ public String getKitRequestIdByBSPSampleId(String bspSampleId) {
         });
 
         if (results.resultException != null) {
-            throw new RuntimeException("Couldn't get kit request id for " + bspSampleId, results.resultException);
+            throw new RuntimeException("Couldn't get kit label for kit " + dsmKitRequestId, results.resultException);
         }
-        return (String) results.resultValue;
+        String kitLabel = String.valueOf(results.resultValue);
+        log.info("Got " + kitLabel + " sequencing kit label in DSM DB for " + dsmKitRequestId);
+        return kitLabel;
     }
 
-    public String getKitLabelFromDsmKitRequestId(long dsmKitRequestId) {
-        SimpleResult results = inTransaction((conn) -> {
+    /**
+     * Resamples a kit for a participant with a legacy short ID. The request is for resampling a kit from Pepper Guid and Hruid to their
```

Comment:
What does "resample" mean here?  I have heard this process described in GP as "repatienting".  It might be better to describe this without resampling or repatienting, and focus the naming and comments on which values are changing.